client/src/pages/collections/Create.tsx

```diff
@@ -207,6 +220,20 @@ const CreateCollection: FC<CollectionCreateProps> = ({ handleCreate }) => {
             setAutoID={changeIsAutoID}
           />
         </fieldset>
+
+        <fieldset className={classes.fieldset}>
+          <legend>{collectionTrans('consistency')}</legend>
+          <CustomSelector
+            wrapperClass={classes.select}
+            options={CONSISTENCY_LEVEL_OPTIONS}
+            onChange={(e: React.ChangeEvent<{ value: unknown }>) => {
+              setConsistencyLevel(e.target.value as number);
+            }}
+            value={consistencyLevel} // 1(Session) by default
```

Comment:
Normally comment default value at usestate. 
And if we change the enum value to string, i don't think we need this.